=== modified file 'bin/addons/base/rng/view.rng'
--- bin/addons/base/rng/view.rng	2010-08-12 17:45:47 +0000
+++ bin/addons/base/rng/view.rng	2010-10-29 15:01:04 +0000
@@ -270,6 +270,7 @@
             <rng:optional><rng:attribute name="rowspan"/></rng:optional>
             <rng:optional><rng:attribute name="align"/></rng:optional>
             <rng:optional><rng:attribute name="filters"/></rng:optional>
+            <rng:optional><rng:attribute name="default_focus"/></rng:optional>
             <rng:zeroOrMore>
                 <rng:choice>
                     <rng:ref name="form"/>
@@ -367,6 +368,7 @@
             <rng:optional><rng:attribute name="position"/></rng:optional>
             <rng:optional><rng:attribute name="context"/></rng:optional>
             <rng:optional><rng:attribute name="confirm"/></rng:optional>
+            <rng:optional><rng:attribute name="default_focus"/></rng:optional>
             <rng:zeroOrMore>
                 <rng:choice>
                     <rng:ref name="form" />


